src/components/ContractForm.js

```diff
@@ -100,10 +100,11 @@ class ContractForm extends Component {
 
     isMandatoryFieldsEmpty = () => {
         const { contract } = this.state;
-        if (!!contract.code && !!contract.dateValidFrom && !!contract.dateValidTo) {
-            return false;
-        }
-        return true;
+
+        return !(!!contract.code &&
+            !!contract.policyHolder &&
+            !!contract.dateValidFrom &&
+            !!contract.dateValidTo);
```

Comment:
```suggestion
        return (!contract.code || !contract.policyHolder || !contract.dateValidFrom || !contract.dateValidTo);
```

Reply:
Thanks, for the suggestion. It works fine.